lib/l10n/intl_messages.arb

```
@@ -5365,5 +5347,45 @@
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
+  },
+  "removeLabelFromEmailSuccessfullyMessage": "Email removed from the \"{labelName}\" label",
+  "@removeLabelFromEmailSuccessfullyMessage": {
+    "type": "text",
+    "placeholders_order": [
+      "labelName"
```

RuboCop (1.82.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)